variables.tf

```diff
@@ -88,8 +88,8 @@ variable "vpc_security_group_ids" {
 
 variable "final_snapshot_identifier" {
   description = "(Optional) The identifier of the final snapshot that is to be created immediately before deleting the cluster. If this parameter is provided, 'skip_final_snapshot' must be false."
-  type        = bool
-  default     = false
+  type        = string
+  default     = ""
```

Comment:
This will not work if `final_snapshot_identifier` is not set, it needs to default to `null`